content/embeds/shard-placement-intro.md

```diff
@@ -0,0 +1,17 @@
+In Redis Enterprise Software (RS), the location of master and slave shards on the cluster nodes can impact the database and node performance.
+Master shards are always hosted separately from their slave shards.
```

Comment:
"Master shards and their corresponding slave shards are always placed on separate nodes."